setup/jetstream/log.go

```diff
@@ -0,0 +1,42 @@
+package jetstream
+
+import (
+	"github.com/nats-io/nats-server/v2/server"
+	"github.com/sirupsen/logrus"
+)
+
+var _ server.Logger = &LogAdapter{}
+
+type LogAdapter struct {
+	entry *logrus.Entry
+}
+
+func NewLogAdapter() *LogAdapter {
+	return &LogAdapter{
+		entry: logrus.StandardLogger().WithField("component", "jetstream"),
+	}
+}
+
+func (l *LogAdapter) Noticef(format string, v ...interface{}) {
+	l.entry.Infof(format, v...)
+}
+
+func (l *LogAdapter) Warnf(format string, v ...interface{}) {
+	l.entry.Errorf(format, v...)
```

Comment:
Looks like there are a few copy and paste mistakes? e.g. `Warnf` logging on `Errorf` and `Fatalf` only on `Errorf`?